_posts/2022-05-14-appsync-graphql-subscriptions.md

```diff
@@ -0,0 +1,308 @@
+---
+layout: post
+title: Realtime application with "@aws_subscribe" directive.
+description: Realtime communication between a web app and a GraphQL API
+date: 2022-05-14
+author: Nadia Reyhani
+image: img/blog/protobuf-ts/protobuf-ts.jpg
+tags: ["AWS Appsync", "Graphql", "tutorial", "subscription", "Real-time data", "aws directive"]
+---
+
+![aws-appsync](/img/blog/appsync-subscription/aws-appsync.png)
+
+### An Introduction
+
+In a [Client-Server](https://en.wikipedia.org/wiki/Client%E2%80%93server_model) Architecture, a client is a piece of software that accesses a specific service or a resource made available by a server through the HTTP protocol in the form of a URL and receives a response. A server On the other hand is a type of computer that receives the request, stores or processes it, and sends response information to the client through the HTTP/HTTPS protocol.
+
+API that stands for `Application Programming Interface` is a software intermediary that allows two applications to talk to each other. APIs provide a standardized way for two applications to send data back and forth.
```

Comment:
`Application Programming Interface` is a software intermediary, should be  `Application Programming Interface` and is a software intermediary